### ISSUE TITLE
Feature Request:  option "Force Speaker" for alarms

### ISSUE THREAD

**Issue by fezulin** [nightscout/AndroidAPS#2500]
In xDrip+ there is the possibility to select a "Force Speaker" option for a "Glucose Level Alarm". 
Even if a headset is connected, the alarm then comes through the speaker of the cell phone. 
Background:
If I listened to music or made a phone call using a headset and then put the headset down, then an alarm from AAPS is only heard through the headset. But because I have already put down the headset, I sometimes cannot hear an alarm. 
In short: 
It would be nice to be able to define the alarms from AAPS with the "Force Speaker" option as well.

Edit:
Observed with AAPS [ 3.1.0.3] with Google Pixel 4.
